component/site/libraries/registry.php

```diff
@@ -11,132 +11,134 @@
  */
```

Comment:
Are changes to this file only formatting?  I can't see what has changed very clearly.